packages/tpetra/core/test/CrsGraph/CrsGraph_UnitTests_Swap.cpp

```diff
@@ -125,55 +125,58 @@ TEUCHOS_STATIC_SETUP()
 // gbl_num_columns: Max # of columns in the matrix-representation of the graph.
 //                  This should be >= the highest value of v from all edges (u,v) in edges.
 //                  Note: u and v are 0-indexed, so if the highest v is 11, then this should be 12.
-template<class LO, class GO, class Node, class comm_t>
+template<class LO, class GO, class Node, class comm_type>
 Teuchos::RCP<Tpetra::CrsGraph<LO, GO, Node>>
-generate_crsgraph(Teuchos::RCP<comm_t>                      comm,
+generate_crsgraph(Teuchos::RCP<comm_type>                      comm,
                   const std::vector<std::pair<GO, GO>>&     gbl_edges,
                   const std::vector<std::pair<GO, size_t>>& gbl_row_owners,
                   const size_t                              gbl_num_columns,
                   const bool                                do_fillComplete=true)
 {
     using Teuchos::Comm;
 
-    typedef Tpetra::CrsGraph<LO, GO, Node> graph_t;
-    typedef Tpetra::Map<LO, GO, Node>      map_t;
+    typedef Tpetra::CrsGraph<LO, GO, Node> graph_type;
```

Comment:
For future reference, prefer `using` syntax as the One Syntax for type aliases.  For example:
```c++
using graph_type = Tpetra::CrsGraph<LO, GO, Node>;
template<class GO>
using my_graph_type = Tpetra::CrsGraph<int, GO>;
```
We prefer `using` syntax because it makes type aliases read just like variable assignment.  "Define `new_name_type` to be Something."  I say "for future reference," because plenty of Tpetra code uses C++98 `typedef`.

Reply:
Updated to use the `using` syntax.